_includes/payment-resource.md

```diff
@@ -44,7 +45,7 @@ Content-Type: application/json
         "payerReference": "AB1234",
         "initiatingSystemUserAgent": "PostmanRuntime/3.0.1",
         "userAgent": "Mozilla/5.0...",
-        "language": "nb-NO",
+        "language": "sv-SE",
```

Comment:
Hm, I just discovered `language` here as well. It should go hand in hand with `currency`, so perhaps we should get rid of the `currency` parameter and just use the `documentation_section` instead?